README.md

```diff
@@ -1,38 +1,49 @@
 # Lightstep OpenTelemetry Examples
 
-This repo contains example code and resources for working with OpenTelemetry.
+This repository contains example code and resources for configuring a variety of languages with OpenTelemetry. The directory structure for each language looks like this:
 
+```
+go
+├── launcher - these applications are using the opentelemetry launcher, a config layer built
+               on top of opentelemetry
+├── opentelemetry - these applications are configured and instrumented using opentelemetry
```

Comment:
FFTI - thoughts about calling these opentelemetry_launcher and opentelemetry_raw or something like that?  Wondering if someone browsing here will think they should be looking at the "opentelemetry" one rather than "launcher".